docusaurus/docs/dev-docs/migration/v4-to-v5/breaking-changes/i18n-content-manager-locale.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Internationalization (i18n)  is now part of the strapi core
+description: Internationalization (i18n) is now part of the Strapi core and no longer a plugin, and this impacts how some the locale parameter is sent and accessed.
+sidebar_label: i18n part of strapi core
+displayed_sidebar: devDocsMigrationV5Sidebar
+tags:
+ - breaking changes
+ - i18n
+---
+
+import Intro from '/docs/snippets/breaking-change-page-intro.md'
+import MigrationIntro from '/docs/snippets/breaking-change-page-migration-intro.md'
+
+# Internationalization (i18n) is now part of the strapi core 
+
+Internationalization (i18n) is now part of the Strapi core and no longer a plugin, and this impacts how some parameters are sent and accessed.
+
+ <Intro />
+
+## Breaking change description
+
+<SideBySideContainer>
+
+<SideBySideColumn>
+
+**In Strapi v4**
+
+Content Manager sends the following parameter when accessing a locale: `plugins[i18n][locale]`.
+
+</SideBySideColumn>
+
+<SideBySideColumn>
+
+**In Strapi v5**
+
+The parameter sent to the Content Manager is now just `locale`.
```

Comment:
I believe this could be marked as an "internal" breaking change, as this is something not normally exposed to users. 

This change is not really user-facing, but it could be important for folks who deeply customize strapi.

I would personally not put it at the same level as the other breaking changes.

The docs look good !

Reply:
🤔 Probably it could be great if there weren't many internal breaking changes, so maybe it's not worth mentioning this internal change.

I originally liked how react flow ( a react library) [presents its breaking changes](https://github.com/xyflow/xyflow/discussions/3764#:~:text=instead%20of%200-,Internal%20changes,-These%20changes%20are). They have a `Internal changes` section where they mention things like this.

But yeah.. If we do that we should be documenting everything that we are breaking internally, and that might be too much :/ So if docs.strpai.io is for user facing things, maybe I would exclude this point, and remove it from our notion db too.

Reply:
Thanks for the suggestion, Marc! I'll keep it for now and we'll see how things go over time :)

Reply:
I've just pushed it to an "internal changes" section. Thank you for the recommendation, Marc, very good idea!
(see [commit](https://github.com/strapi/documentation/pull/1991/commits/476d74cb947828f7fd6c5baacef4c38652a74f16) and [live preview](https://documentation-git-v5-breaking-changes-i18n-move-d05ddf-strapijs.vercel.app/dev-docs/migration/v4-to-v5/breaking-changes#internal-changes))